lib/db/schema/patch-080-081.sql

```diff
@@ -0,0 +1,59 @@
+SET NAMES utf8mb4 COLLATE utf8mb4_bin;
+
+CREATE TABLE IF NOT EXISTS recoveryKeys (
+  uid BINARY(16) NOT NULL,
+  recoveryKeyId BINARY(64) NOT NULL,
+  recoveryData TEXT,
+  UNIQUE KEY (`uid`, `recoveryKeyId`)
+) ENGINE=InnoDB;
+
+CREATE PROCEDURE `createRecoveryKey_1` (
+  IN `uidArg` BINARY(16),
+  IN `recoveryKeyIdArg` BINARY(64),
+  IN `recoveryDataArg` TEXT
+)
+BEGIN
+
+  DECLARE EXIT HANDLER FOR SQLEXCEPTION
+  BEGIN
+    ROLLBACK;
+    RESIGNAL;
+  END;
+
+  START TRANSACTION;
+
+  SET @accountCount = 0;
+
+  -- Signal error if no user found
+  SELECT COUNT(*) INTO @accountCount FROM `accounts` WHERE `uid` = `uidArg`;
+  IF @accountCount = 0 THEN
+    SIGNAL SQLSTATE '45000' SET MYSQL_ERRNO = 1643, MESSAGE_TEXT = 'Can not create recovery key for unknown user.';
+  END IF;
+
+  INSERT INTO recoveryKeys (uid, recoveryKeyId, recoveryData) VALUES (uidArg, recoveryKeyIdArg, recoveryDataArg);
+
+  COMMIT;
+END;
+
+CREATE PROCEDURE `getRecoveryKey_1` (
+  IN `uidArg` BINARY(16),
+  IN `recoveryKeyIdArg` BINARY(64)
+)
+BEGIN
+
+  SELECT * FROM recoveryKeys WHERE uid = uidArg AND recoveryKeyId = recoveryKeyIdArg;
```

Comment:
As a purely stylistic thing, I think it's better to explicitly list columns than doing `select *`, it reduces the change of accidentally leaking newly-added columns in places you didn't intend.